intro-HPC/gent/macros.tex

```diff
@@ -3,10 +3,11 @@
 \macro{\university}{UGent}
 \macro{\association}{AUGent}
 % HPC definitions
-\macro{\hpc}{HPC-UGent}
-\macro{\hpcname}{Delcatty}
+\macro{\hpc}{UGent-HPC}
```

Comment:
we have a dedicated issue for converging on this first, so please don't change it until it's settled, roll back to HPC-UGent